auth/store_test.go

```diff
@@ -213,3 +217,191 @@ func TestUserGrant(t *testing.T) {
 		t.Fatalf("expected %v, got %v", ErrUserNotFound, err)
 	}
 }
+
+func TestGetUser(t *testing.T) {
+	as, tearDown := setupAuthStore(t)
+	defer tearDown(t)
+
+	_, err := as.UserGrantRole(&pb.AuthUserGrantRoleRequest{User: "foo", Role: "role-test"})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	u, err := as.UserGet(&pb.AuthUserGetRequest{Name: "foo"})
+	if err != nil {
+		t.Fatal(err)
+	}
+	if u == nil {
+		t.Fatalf("expected user object, got %v", err)
+	}
+	expected := []string{"role-test"}
+	if !reflect.DeepEqual(expected, u.Roles) {
+		t.Fatalf("expected %v, got %v", expected, u.Roles)
+	}
+}
+
+func contains(array []string, str string) bool {
+	set := make(map[string]struct{}, len(array))
```

Comment:
Do we need to create the map? How about just iterating `array` and compare its element with `str`?